todo/src/style/index.css

```
@@ -0,0 +1,36 @@
+.bg-High {
+    background-color: var(--bg-high)!important;
+}
+
+.bg-Medium {
+    background-color: var(--bg-medium)!important;
+}
+
+.bg-Low {
+    background-color: var(--bg-low)!important;
+}
+
+* {
+    transition: background-color 1s ease-in;
+}
+
+body.dark {
+    background-color: rgba(0, 0, 0, 0.801);
+    color:rgba(255, 255, 255, 0.712)
+}
+
+body.dark .modal-content {
+    background-color:rgba(24, 22, 22, 0.9) ;
+}
+
+body.light {
+    background-color: rgba(0, 0, 0, 0.178);
+}
+
+body.dark .navbar {
+    background-color: rgba(0, 0, 0, 0.178) !important;
```

use css variables for this too

---

Now I use css variables here in other place.
```suggestion
body {
    color: var(--text-black);
    & .bg-high {
        background-color: var(--bg-light-high-priority);
    }
    
    & .bg-medium {
        background-color: var(--bg-light-medium-priority);
    }
    
    & .bg-low {
        background-color: var(--bg-light-low-priority);
    }
}
```